vb-fe/src/util/User.ts

```diff
@@ -0,0 +1,12 @@
+export interface ILocalStorageUser {
```

Comment:
whats the difference between this and the User interface that we will have, or is this the user interface? If so i think you should just called it `User`

Reply:
I dont think we currently have a user interface

Reply:
Ah yeah we don't. Rename this to User and import it in other places (eg. `host` and `user` (User[]) in the room interface)

Reply:
or UserType. I'm down to calling all generic interfaces <NAME>Type. That should tell us its an interface for the particular entity